mp/src/game/client/momentum/ui/HUD/hud_speedometer.cpp

```diff
@@ -199,6 +203,11 @@ void CHudSpeedMeter::OnThink()
             vel *= 0.04262f;
             lastJumpVel *= 0.04262f;
             SetLabelText(L"MPH");
+            break;
+		case 4:
+            // Normalized units of energy
+            vel = ( (pow(velocity2.x, 2) + pow(velocity2.y, 2) + pow(velocity2.z, 2))/2 + (800.0f * (pPlayer->GetLocalOrigin().z - lastJumpZPos)) ) / 800.0f;
```

Comment:
Use the `Vector::Length()` method rather than manually calculating length like this